_includes/language-picker.html

```diff
@@ -1,14 +1,16 @@
-<div class="language-picker">
+<div class="language-picker{% if page.layout == 'home' %} language-picker--home language-picker--right-orientation{% endif %}">
   <span class="language-picker__current">
-    {{ site.data.translations.header-language-picker-label[page.lang] }}:
+    <span class="language-picker__label">
+      {{ site.data.translations.header-language-picker-label[page.lang] }}:
+    </span>
     <strong>{{ site.data.languages[page.lang].label }}</strong>
   </span>
   <button type="button" class="language-picker__toggle" aria-controls="language-picker__dropdown" aria-expanded="false">
     <span class="screen-reader-text">
       {{ site.data.translations.header-language-picker-switch-lang[page.lang] }}
     </span>
   </button>
-  <div class="language-picker__chevron">&nbsp;</div>
+  <div class="language-picker__chevron"></div>
   <div class="language-picker__dropdown" id="language-dropdown">
     <h3 class="language-picker__title">
```

Comment:
de eerste titel van de pagina is nu geen `<h1>`, maar deze `<h3>`. Voor een goede structuur is het belangrijk dat de eerste titel een `<h1>` is. Je zou dit kunnen oplossen door van dit element een niet-heading te maken, of door 'm in de structuur onder een `<h2>` te laten vallen (of er een `<h2>` van te maken onder de `<h1>`). Ik denk dat dat eerste het beste is in deze context.

Reply:
Veranderd in een span